### DISCUSSION TITLE
Mi pokédex!

### DISCUSSION THREAD

**Discussion by David-HernandezM** [Launch-X-Latam/MisionFrontEnd#392] [Show and tell]
¡Hola!, ¿Cómo están?, quería compartirles mis avances con la pokédex!
Link del repositorio: [repositorio](https://github.com/David-HernandezM/Pokedex)
Página en vivo: [Página](https://david-hernandezm.github.io/Pokedex/)

**Comment by JocelinFlores** [comment]
Esta súper completa, muchas felicidades!
